src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -318,6 +318,29 @@ Da nur unpersönliche Zufallscodes übertragen werden, bleibt Ihre Identität un
 "Home_Risk_Status_Check_Body" = "Es werden aktuelle Daten heruntergeladen und geprüft. Dies kann mehrere Minuten dauern.";
 "Home_Risk_Status_Check_Button" = "Aktualisieren";
 
+// Home Thank you card
+"Home_Thank_You_Card_Title" = "Vielen Dank!";
+"Home_Thank_You_Card_Body" = "Dank Ihrer Mithilfe können andere gewarnt werden und entsprechend reagieren.";
+"Home_Thank_You_Card_Note_Title" = "Bitte beachten Sie:";
+"Home_Thank_You_Card_Phone_Item_Title" = "Das Gesundheitsamt wird sich in den nächsten Tagen telefonisch oder schriftlich bei Ihnen melden.";
+"Home_Thank_You_Card_Home_Item_Title" = "Sie sind ansteckend. Isolieren Sie sich von anderen Personen.";
+"Home_Thank_You_Card_Further_Info_Item_Title" = "Weitere Infos:";
+"Home_Thank_You_Card_14Days_Item_Title" = "Ihre Quarantänezeit beträgt im Regelfall 14 Tage. Beobachten und erfassen Sie genau, wie sich ihre Krankheitszeichen entwickeln.";
+"Home_Thank_You_Card_Contacts_Item_Title" = "Sie werden von Ihrem Gesundheitsamt gebeten, eine Liste Ihrer Kontaktpersonen zu erstellen. Dabei sollen alle Personen erfasst werden, mit denen Sie in den zwei Tagen vor Erkrankungsbeginn engen Kontakt (unter 2 Meter, direktes Gespräch) über insgesamt 15 Minuten hatten.";
+"Home_Thank_You_Card_App_Item_Title" = "Bitte denken Sie hier auch besonders an Personen, die nicht automatisch durch die App informiert werden, da sie kein Smartphone oder die App haben.";
+"Home_Thank_You_Card_NoSymptoms_Item_Title" = "Auch wenn Sie keine Krankheitszeichen mehr haben und sich wieder gesund fühlen, könnten Sie noch ansteckend sein.";
+
+// Home Finding positive
```

Comment:
wrong comment syntax in translation file: please use /* */ instead of //

Reply:
Shouldn't we have one text and translation for home and submission and not separate for both

Reply:
I agree, we have redundant strings here (see the `ExposureSubmissionSuccessViewController`. This will make later adjustments of the text unnecessary hard.

Reply:
yeap ..........

Reply:
I suggest to do this mapping later, changes in this task should be merged asap, so other could do integration this cards. @johanneseschrig @thomasaugsten are you ok with it?